Doesn't compile due to warnings treated as errors.

Just cloned (fc7dddfa5b6fd669c5c2bbbdd58e3413a3c6d145) and this is the result of running build.sh.

```
➜ ./build.sh
mkdir: cannot create directory ‘build’: File exists
./build.sh: line 9: cmake3: command not found
-- Configuring done
-- Generating done
-- Build files have been written to: /home/friela/c/gh/azure-storage-fuse/build
[  3%] Building CXX object CMakeFiles/blobfuse.dir/azure-storage-cpp-lite/src/tinyxml2.cpp.o
/home/friela/c/gh/azure-storage-fuse/azure-storage-cpp-lite/src/tinyxml2.cpp: In static member function ‘static void tinyxml2::XMLUtil::ConvertUTF
32ToUTF8(long unsigned int, char*, int*)’:
/home/friela/c/gh/azure-storage-fuse/azure-storage-cpp-lite/src/tinyxml2.cpp:337:19: error: this statement may fall through [-Werror=implicit-fall
through=]
             input >>= 6;
             ~~~~~~^~~~~
/home/friela/c/gh/azure-storage-fuse/azure-storage-cpp-lite/src/tinyxml2.cpp:338:9: note: here
         case 3:
         ^~~~
/home/friela/c/gh/azure-storage-fuse/azure-storage-cpp-lite/src/tinyxml2.cpp:341:19: error: this statement may fall through [-Werror=implicit-fall
through=]
             input >>= 6;
             ~~~~~~^~~~~
/home/friela/c/gh/azure-storage-fuse/azure-storage-cpp-lite/src/tinyxml2.cpp:342:9: note: here
         case 2:
         ^~~~
/home/friela/c/gh/azure-storage-fuse/azure-storage-cpp-lite/src/tinyxml2.cpp:345:19: error: this statement may fall through [-Werror=implicit-fall
through=]
             input >>= 6;
             ~~~~~~^~~~~
/home/friela/c/gh/azure-storage-fuse/azure-storage-cpp-lite/src/tinyxml2.cpp:346:9: note: here
         case 1:
         ^~~~
cc1plus: all warnings being treated as errors
CMakeFiles/blobfuse.dir/build.make:254: recipe for target 'CMakeFiles/blobfuse.dir/azure-storage-cpp-lite/src/tinyxml2.cpp.o' failed
make[2]: *** [CMakeFiles/blobfuse.dir/azure-storage-cpp-lite/src/tinyxml2.cpp.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/blobfuse.dir/all' failed
make[1]: *** [CMakeFiles/blobfuse.dir/all] Error 2
Makefile:151: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/azure-storage-cpp-lite/src/tinyxml2.cpp
+++ b/azure-storage-cpp-lite/src/tinyxml2.cpp
@@ -334,15 +334,15 @@ void XMLUtil::ConvertUTF32ToUTF8( unsigned long input, char* output, int* length
         case 4:
             --output;
             *output = (char)((input | BYTE_MARK) & BYTE_MASK);
-            input >>= 6;
+            input >>= 6; /* fall-thru */
         case 3:
             --output;
             *output = (char)((input | BYTE_MARK) & BYTE_MASK);
-            input >>= 6;
+            input >>= 6; /* fall-thru */
         case 2:
             --output;
             *output = (char)((input | BYTE_MARK) & BYTE_MASK);
-            input >>= 6;
+            input >>= 6; /* fall-thru */
         case 1:
             --output;
             *output = (char)(input | FIRST_BYTE_MARK[*length]);

